backend/entityservice/tasks/comparing.py

```diff
@@ -186,17 +173,16 @@ def compute_filter_similarity(chunk_info, project_id, run_id, threshold, encodin
     else:
         result_filename = None
         file_size = None
-    t6 = time.time()
+    t5 = time.time()
 
     log.info("run={} Comparisons: {}, Links above threshold: {}".format(run_id, comparisons_computed, len(chunk_results)))
-    log.info("Prep: {:.3f} + {:.3f}, Solve: {:.3f}, Progress: {:.3f}, Offset: {:.3f}, Save: {:.3f}, Total: {:.3f}".format(
+    log.info("Prep: {:.3f} + {:.3f}, Solve: {:.3f}, Progress: {:.3f}, Save: {:.3f}, Total: {:.3f}".format(
         t1 - t0,
         t2 - t1,
         t3 - t2,
         t4 - t3,
-        t4 - t4,
```

Comment:
This line was a bug. Should have said `t5 - t4` the whole time (instead of `t4 - t4`).